src/audio-worklet/migration/worklet-recorder/index.njk

```diff
@@ -23,7 +23,11 @@ to_root_dir: ../../../
 </style>
 
 <h1>AudioWorklet Recorder</h1>
-<p>Demonstrates a simple recording app using Audio Worklet</p>
```

Comment:
I would write:
> This is a simple recording app that uses the AudioWorklet API. The maximum recording time is 5 minutes. After reaching the maximum recording time, the recording will be stopped automatically and a download link for the `.wav` file will be prepared. Due to the file size and recording length, it may take a few moments to prepare the download link and preview the recording. To restart the recording, please click the Refresh button.